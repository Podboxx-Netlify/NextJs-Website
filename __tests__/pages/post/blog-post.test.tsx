import React from 'react'
import {shallow} from 'enzyme'
import BlogPost from '../../../pages/post/[id]'
import {mockNextUseRouter} from "../../../__mocks__/routerMock";

describe('BlogPost', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<BlogPost/>);
    })

    it('should render without crashing', () => {
        expect(wrapper).toMatchSnapshot();
    })

    it('Page should be loading', () => {
        mockNextUseRouter({
            route: "/post/667",
            pathname: "/post/[id]",
            query: {"id": "667"},
            asPath: "/post/667",
        });
        expect(wrapper.contains(<div className="cover-spin" id="cover-spin"/>)).toEqual(true);
    })
})