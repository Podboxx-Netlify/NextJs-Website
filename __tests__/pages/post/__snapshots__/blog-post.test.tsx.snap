// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost should render without crashing 1`] = `
<div
  className="cover-spin"
  id="cover-spin"
/>
`;
